File: src/main/java/org/taktik/icure/dto/filter/patient/PatientByHcPartyAndActiveFilter.java
package org.taktik.icure.dto.filter.patient;

import org.taktik.icure.dto.filter.Filter;
import org.taktik.icure.entities.Patient;

public interface PatientByHcPartyAndActiveFilter extends Filter<String, Patient> {
    boolean getActive();
    String getHealthcarePartyId();
}

File: src/main/kotlin/org/taktik/icure/domain/filter/patient/PatientByHcPartyAndActiveFilter.kt
package org.taktik.icure.domain.filter.patient

import org.taktik.icure.domain.filter.Filter
import org.taktik.icure.entities.Patient

interface PatientByHcPartyAndActiveFilter : Filter<String, Patient> {
    val active: Boolean
    val healthcarePartyId: String?
}
